docs/integrations/sources/file.md

```diff
@@ -0,0 +1,98 @@
+# Files
+
+## Overview
+
+File are often exchanged or published in various remote locations. This source aims to support an expanding range of file formats and storage providers. The File source supports Full Refresh syncs. That is, every time a sync is run, Airbyte will copy all rows in the file and columns you set up for replication into the destination in a new table.
+
+### Output schema
+
+At this time, this source produces only a single stream for the target file as it replicates only one file at a time for the moment. We'll be considering to improve this behavior by globing folders or use patterns to capture more files in the next iterations as well as more file formats and storage providers.
+Note that you should provide the `dataset_name` which dictates how the table will be identified in the destination (since `URL` can be made of complex characters)
+
+### Features
+
+| Feature | Supported? |
+| :--- | :--- |
+| Full Refresh Sync | Yes |
+| Incremental Sync | No |
+| Replicate Incremental Deletes | No |
+| Replicate Folders \(multiple Files\) | No |
+| Replicate Glob Patterns \(multiple Files\) | No |
+
+### Storage Providers
+
+| Storage Providers | Supported? |
+| :--- | :--- |
+| HTTPS | Yes |
+| Google Cloud Storage | Yes |
+| Amazon Web Services S3 | Yes |
+| SFTP / SSH / SCP | Yes |
```

Comment:
We should separate SFTP from SSH / SCP, it is not the same protocol

Reply:
all of those are handled by the same library [paramiko](http://docs.paramiko.org/en/stable/) with the exact same function call that' s why it s regrouped but i can split them and add a little more docs about what is under the hood

Reply:
Yep, that would be great.